src/Request/ConsoleRequest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Bugsnag\BugsnagLaravel\Request;
+
+use Bugsnag\Request\RequestInterface;
+
+class ConsoleRequest implements RequestInterface
+{
+    /**
+     * The unformated console command.
+     *
+     * @var array
+     */
+    protected $command;
+
+    /**
+     * Create a new console request instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->command = $_SERVER['argv'];
```

Comment:
If we do want to go laravel specific, we should never access "super globals" in library files.